shared/src/business/utilities/formatAttachments.ts

```diff
@@ -27,6 +27,7 @@ export const formatAttachments = ({
         archived: !!doc.archived,
         documentId,
         documentTitle: generatedDocumentTitle || doc.documentType,
+        index: doc.index,
```

Comment:
Doesn't need to happen before going to `test`, but we'll probably need updated tests on this utility and the action below